isis/src/control/objs/BundleAdjust/BundleAdjust.cpp

```diff
@@ -495,6 +495,30 @@ namespace Isis {
 
     if (m_bundleSettings->solveTargetBody()) {
       m_rank += m_bundleSettings->numberTargetBodyParameters();
+
+      if (m_bundleTargetBody->solveMeanRadius()){
+        // Check if MeanRadiusValue is abnormal compared to observation
+        bool isMeanRadiusValid = true;
+        double localRadius, aprioriRadius;
+
+        // Arbitrary control point containing an observed localRadius
+        BundleControlPointQsp point = m_bundleControlPoints.at(0);
+        SurfacePoint surfacepoint = point->adjustedSurfacePoint();
+
+        localRadius = surfacepoint.GetLocalRadius().meters();
+        aprioriRadius = m_bundleTargetBody->meanRadius().meters();
+
+        // Ensure aprioriRadius is within some threshold
+        // Considered potentially inaccurate if it's off by atleast a factor of two
+        if (aprioriRadius >= 2 * localRadius || aprioriRadius <= localRadius / 2) {
+            isMeanRadiusValid = false;
+        }
+
+        // Warn user for abnormal MeanRadiusValue
+        if (!isMeanRadiusValid) {
+          outputBundleStatus("Warning: User-entered MeanRadiusValue may be inaccurate\n");
```

Comment:
I think we may want a slightly stronger warning here. Something like, "This option may introduce inaccuracy into the overall bundle adjustment." Still a solid change overall!

Reply:
I added a warning that outputs for any target body radii solve, and added some more detail on the warning for inaccurate mean radius input.